src/main/java/com/sumindaa/demo/employeeservice/controller/EmployeeController.java

```diff
@@ -28,14 +28,16 @@ public Employee getEmployee(@PathVariable String id) {
     public void addEmployee(@RequestBody Employee employee) {
         employeeService.addEmployee(employee);
     }
+
+
+    @RequestMapping(method = RequestMethod.PUT, value = "/employees/{id}")
+    public void addEmployee(@RequestBody Employee employee, @PathVariable String id) {
+        employeeService.updateEmployee( id, employee);
+    }
```

Comment:
why add employee need an id?

Reply:
otherwise how do we identify the particular employee